### PR TITLE
INTERACTIV-list

### PR DESCRIPTION
**Pull request by Abel-desu**
**IN THIS PROJECT I USED :-** 

- eventlistiner
- add and remove functionality
- webpack
- and basic javascript
- toggle

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -77,12 +77,31 @@ const createList = () => {
...
+  for (let i = 0; i < variable.length; i += 1) {
+    form.removeChild(variable[i]);
+  }
+  const empty = [];
+  for (let i = 0; i < getting.length; i += 1) {
+    if (getting[i].completed === true) {
+      continue;
+    }
+    empty.push(getting[i]);
+  }
```
Comment (AshakaE) [inline src/index.js]: - [ ] Kindly implement this loop using the `Array.filter()` method

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @Abel-desu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -77,12 +77,31 @@ const createList = () => {
...
+// clear all
+const clearAll = document.querySelector('.clear');
+clearAll.addEventListener('click', () => {
+  const getting = JSON.parse(localStorage.getItem('list'));
+  const variable = document.querySelectorAll('.change');
+  for (let i = 0; i < variable.length; i += 1) {
+    form.removeChild(variable[i]);
+  }
+  const empty = [];
+  for (let i = 0; i < getting.length; i += 1) {
```
Comment (topeogunleye) [inline src/index.js]: - [ ] As suggested by the previous code reviewer and project requirements. I kindly suggest that you make use of the filter method for your "clearAll" function

##  EXAMPLE
```
export default function clearAll(data = getDataLS()) {
  data = data.filter((task) => !task.completed);
  localStorage.setItem('tasks', JSON.stringify(data));
  indexValue();
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Abel-desu,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Ismailco — APPROVED** [review]
### Hi @Abel-desu,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**